packages/admin/admin-icons/generate-icons.ts

```diff
@@ -69,9 +69,29 @@ const getSVGData = (icon: Icon) => {
             delete attrs["@_fill"];
             return true;
         },
-    }).parse(fileContents.toString());
+    });
+    const parsed = parsedXml.parse(fileContents.toString());
+
+    return convertAttributesToCamelCase(parsed.svg);
+};
+
+const convertAttributesToCamelCase = (node: any): any => {
+    if (Array.isArray(node)) {
+        return node.map(convertAttributesToCamelCase);
+    }
+
+    if (typeof node !== "object" || node === null) {
+        return node;
+    }
+
+    const result: any = {};
+
+    for (const [key, value] of Object.entries(node)) {
+        const camelCaseKey = key.replace(/-([a-z])/g, (_, char) => char.toUpperCase());
+        result[camelCaseKey] = convertAttributesToCamelCase(value);
+    }
```

Comment:
Wouldn't it be better to perform this in [updateTag](https://github.com/vivid-planet/comet/pull/3853/files#diff-b049f07729e8033ea3cfc4e5fbb5dc3bcc25679b258433a9f5eedb60b1730091R68)?

Reply:
yeah true, moved to logic into the updateTag function